status-storage-spec.md

```diff
@@ -0,0 +1,9 @@
+# Status Storage Specification
```

Comment:
I'd also call these e.g. IPFS usage and what not, it isn't a storage spec. Ditto blockchain (Ethereum use?)